src/ngx_http_auth_jwt_module.c

```diff
@@ -442,11 +442,17 @@ loadAuthKey(ngx_conf_t *cf, ngx_http_auth_jwt_loc_conf_t* conf) {
 	}
 
 	conf->_auth_jwt_keyfile.data = ngx_palloc(cf->pool, keySize);
-	fread(conf->_auth_jwt_keyfile.data, 1, keySize, keyFile);
-	conf->_auth_jwt_keyfile.len = (int)keySize;
+	if(fread(conf->_auth_jwt_keyfile.data, 1, keySize, keyFile)) {
```

Comment:
Should check if the result of `fread(...)` is the same as `keySize`, since it returns the number of elements read. That is, let's make sure we read the expected number of elements/bytes rather than what the `if` does now, which would pass if at least one byte was read.